cocos/core/math/quat.ts

```diff
@@ -250,6 +265,7 @@ export class Quat extends ValueType {
     }
 
     /**
+     * @en The component w xyz components calculated, normalized by default
```

Comment:
Calculates the w component with xyz components, considering the given quaternion normalized